packages/gestalt/src/Colors.css

```diff
@@ -40,6 +40,14 @@
   background-color: var(--gestalt-colorRed100);
 }
 
+.red0 {
```

Comment:
where is this used?

Reply:
That one isn't currently used. I added it for completeness. I'll remove it since it's un-needed for now